aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -2493,8 +2493,6 @@ void quantize_tensor_per_tensor_affine_cpu(
 #if defined(__ARM_NEON__) || defined(__aarch64__)
   AT_DISPATCH_QINT_TYPES(
       qtensor.scalar_type(), "quantize_tensor_per_tensor_affine_cpu", [&]() {
-        TORCH_CHECK(
-            rtensor.is_contiguous(), "Float tensor should be contiguous");
         const float* const rdata = rtensor.data_ptr<float>();
```

Comment:
I don't understand this.  It's usually not safe to call `data_ptr` on a non-contiguous tensor.  Maybe the check should be that rtensor is contiguous in the memory format of qtensor?  And what if qtensor is not contiguous?

Reply:
Oh yes. Thanks for the catch. Dont know what was I thinking removing this, maybe it will come back.